### DISCUSSION TITLE
Anton Petrov on "Evidence That Some JWST Galaxies May Instead Be Something Exotic"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#27] [Astrophysics and Physics]
Two weeks ago Anton Petrov posted a [video claiming that JWST galaxies were explained by the *Renaissance* simulation](https://youtu.be/yjL45bxX4po) of the early universe.  The [work published on ArXiV](https://arxiv.org/abs/2304.13755) uses a statistically **rare** occurrence of developed galaxies in the simulation to claim that it explains JWST observations.
Indeed it explains observations with the corresponding statistically low degree of confidence - in other words it doesn't explain anything but a fluke in the simulation.

This other video published a week ago, [Evidence That Some JWST Galaxies May Instead Be Something Exotic](https://youtu.be/07ZLW6ygaHg), proposes another explanation for the galaxies observed by JWST (needed because the *Renaissance* explanation is worthless...)

The explanation (one of many others that don't work) is based on the hypothesis of *dark stars*, a dense accumulation of dark matter with some hydrogen and helium gas orbiting around the dark star.

To make this hypothesis work one needs to assume:
1. that dark matter (DM) exists,
1. that DM is composed of particles,
1. DM particles can self-annihilate, producing energy that's released as light (now it interacts with light?),
1. the dark stars are supermassive (how can that be so early after the Big Bang?),
1. the composite object is DM at its centre and baryonic matter surrouding it (H, He, and *possibly* other matter),
1. the gas cloud somehow remains at low density so there is no fusion,
1. the released energy prevents the collapse of the cloud (but why not for other clouds that form galaxies?),
1. the numerous previous assumptions are required to explain galaxy JADES-GS-z13-0 and three others (overfitting),
1. that another model (Population-III stars) can co-exist and explain early stars after the Big Bang,
1. that the observed Lyman-break in the spectrum does not come from regular galaxies,
1. that the observed HeII spectral lines are not from Pop-III stars (despite seeing [Pop-III signatures in other galaxies](https://arxiv.org/abs/2306.00953)).

The large number of assumptions to make this work is a sign of very bad science.  The article, as described by Anton Petrov, is not credible at all.

P.S.  JADES-GS-z13-0 and other galaxies look small because [at redshift z = 13.2 their distance is 35 Gly](http://www.marmet.org/cosmology/calculator/index.html).

This is a followup to the post by Mitchell on May 29th
> Hey everyone, I've noticed that both Dr. Becky and Anton Petrov, two youtubers running science channels, have put out videos 
that purport to explain away the too massive and old galaxies observed in the early universe. We already discussed Becky's video recently, but Anton recently put out some videos of his own. *[...]*

> This video brings up the new simulations claiming that the JWST galaxies are in agreement with LCDM [Major Explanation For JWST Galaxies That Broke Modern Theories](https://youtu.be/yjL45bxX4po)

**Comment by Pierre314159** [comment]
What did they say, again, about moderation on arXiv being too strict?
